src/adaptor/native.ts

```diff
@@ -0,0 +1,30 @@
+import nativeFirestore, {
+  FirebaseFirestoreTypes
+} from '@react-native-firebase/firestore'
+// @ts-ignore: React Native Firebase doesn't export types for internal modules
+import FirestoreDocumentReference from '@react-native-firebase/firestore/lib/FirestoreDocumentReference'
+// @ts-ignore
+import FirestoreFieldValue from '@react-native-firebase/firestore/lib/FirestoreFieldValue'
+// @ts-ignore
+import FirestoreTimestamp from '@react-native-firebase/firestore/lib/FirestoreTimestamp'
+import { getAll } from './utils'
+
+const DocumentReference = FirestoreDocumentReference as FirebaseFirestoreTypes.DocumentReference
+const Timestamp = FirestoreTimestamp as FirebaseFirestoreTypes.Timestamp
+const FieldValue = FirestoreFieldValue as FirebaseFirestoreTypes.FieldValue
+
+export default async function adaptor() {
+  const firestore = nativeFirestore()
+  // At the moment, the React Native Firebase adaptor doesn't support getAll.
```

Comment:
Could you elaborate on this a bit? I'm not sure what is the purpose of this getAll, maybe I could suggest something that would make it work

Reply:
Oh, it's actually not a problem. Node.js's Firestore supports this function while browser and React Native doesn't. The polyfill works perfectly fine.

See: https://cloud.google.com/nodejs/docs/reference/firestore/0.15.x/Firestore#getAll